Remove -gstabs, which is not supported by clang.

The configure for jimctl wants to add ccache in front of CC by default, which
ends up as 'cccache ccache ${configure.cc}' for MacPorts. Disable automatic use
of ccache.
--- jimtcl/autosetup/cc.tcl.orig	2013-02-07 17:51:18.000000000 +0100
+++ jimtcl/autosetup/cc.tcl	2013-02-07 17:54:37.000000000 +0100
@@ -493,12 +493,6 @@
 	}
 	lappend cmdline {*}$opts(-cflags)
 
-	switch -glob -- [get-define host] {
-		*-*-darwin* {
-			# Don't generate .dSYM directories
-			lappend cmdline -gstabs
-		}
-	}
 	lappend cmdline $src -o $tmp {*}$opts(-libs)
 
 	# At this point we have the complete command line and the
@@ -668,7 +662,7 @@
 	user-error "Could not find a C compiler. Tried: [join $try ", "]"
 }
 
-define CCACHE [find-an-executable [get-env CCACHE ccache]]
+define CCACHE {}
 
 # Initial cctest settings
 cc-store-settings {-cflags {} -includes {} -declare {} -link 0 -lang c -libs {} -code {}}
